src/plugins/console/server/lib/spec_definitions/json/generated/nodes.usage.json

```diff
@@ -19,6 +19,6 @@
         "rest_actions"
       ]
     },
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/cluster-nodes-usage.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/index/#clusters-and-nodes"
```

Comment:
Same thing here - this documentation is not for the actual API. Add to `noDocumentation`